MovieAppPetProject/MovieAppPetProject/HomeScreen/HomeScreenLogic/MoviesViewModel.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  NowPlayingViewModel.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/04/08.
+//
+
+import Foundation
+
+// MARK: - Protocol
+protocol MoviesViewModelType: AnyObject {
+    func reloadView()
+}
+
+class MoviesViewModel {
+
+    // MARK: - Variables
+    var movies: [Movie]?
```

Comment:
```suggestion
    private var movies: [Movie]?
```

Please make this property private because we don't want it to be changed in the view controller. 

Please either use computed properties or functions to get the info you need in the view model and make that public. Message me if you need some more clarity

Reply:
will add computed properties